src/main/java/uk/gov/hmcts/darts/cache/token/component/impl/TokenValidatorImpl.java

```diff
@@ -95,7 +98,7 @@ boolean validateTheTokenExpiry(String accessToken) {
             JWT jwt = SignedJWT.parse(accessToken);
             Object expired = jwt.getJWTClaimsSet().getClaim("exp");
             if (expired != null) {
-                long expiry = ((Date) expired).getTime() - EXPIRE_JWT_MINUTES_BEFORE_TOKEN_EXPIRY * (1000 * 60);
+                long expiry = ((Date) expired).getTime() - cacheProperties.getShareTokenAllocationEarlyExpirationMinutes() * (1000 * 60);
```

Comment:
observation - took me a while to work out that expiry is in milliseconds! may be worth renaming. mind you, i think i am getting PR fatigue